functions/index.js

```diff
@@ -116,19 +118,66 @@ async function snapshotData(filename, html_snippet_filename) {
   return [data, html_snippets];
 }
 
+// Fetch lat & lng for the given address by making a call to the Google Maps API.
+// Returns a promise.
+async function getLatLng(address, client) {
+  return client
```

Comment:
nit...you're using a promise in an async function. I think you could have just done
const r = await client.geocode(...);
if (r.data.results & r.data.results.length > 0) {
  ....
}

Reply:
Done.